cypress/e2e/cloud/globalHeader.test.ts

```diff
@@ -243,6 +242,12 @@ describe('change-account change-org global header', () => {
   })
 
   describe('user profile avatar', {scrollBehavior: false}, () => {
+    before(() => {
```

Comment:
Improves consistency of the user profile icon appearing in the header in this test.